lib/helpers.coffee

```diff
@@ -31,7 +31,9 @@ module.exports = Helpers =
         else
           resolve(data.stderr.join(''))
       if isNodeExecutable
-        spawnedProcess = new BufferedNodeProcess({command, args, stdout, stderr, exit})
+        options.env = JSON.parse(JSON.stringify(process.env))
```

Comment:
It doesn't return an empty object, it just moves stuff to the prototype. You might not see it in the debugger at top level but it's still there.